.github/workflows/prod.yaml

```diff
@@ -43,9 +43,14 @@ jobs:
   howl-on-fail:
     runs-on: ubuntu-latest
     needs: [deploy-stage, release, deploy-prod]
-    if: failure()
+    if: always() && contains(join(needs.*.result, ','), 'failure')
     steps:
-      - uses: foxygoat/howl@v1
+      - uses: actions/checkout@v4
+      - run: ./bin/howl
         env:
-          SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
-          SLACK_TEXT: <!here|here>
+          # Notify foxygoat Slack workspace on channel #gh-notify
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
I think you should only send to one place. It's annoying to get multiple notifications for the same thing. I would do discord only.

Reply:
ok.